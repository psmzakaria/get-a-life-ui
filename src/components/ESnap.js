import React from "react";
import { Card } from "semantic-ui-react";

const ESnap = props => {
  return (
    <div className="esnap-container-size">
      <Card color="red">
        {/* <div className="esnap-icon">
            <Icon name="calendar alternate outline" size="massive" />
          </div> */}
        <Card.Content>
          <Card.Header>{props.title}</Card.Header>
          <Card.Meta>
            <span>Host: {props.host}</span>
          </Card.Meta>
          <Card.Description>KIV - information/details</Card.Description>
        </Card.Content>
        <Card.Content extra>
          <a>{props.status}</a>
        </Card.Content>
      </Card>
    </div>
  );
};

export default ESnap;
