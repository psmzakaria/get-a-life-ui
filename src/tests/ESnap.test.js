import React from "react";
import ESnap from "./../components/ESnap";

import ShallowRenderer from "react-test-renderer/shallow";

it("renders the create component of the app ", () => {
  const renderer = new ShallowRenderer();
  const event = {
    title: "birthday",
    hostId: {
      username: "lala"
    },
    proposedDates: []
  };

  renderer.render(<ESnap key={1} event={event} />);
  const output = renderer.getRenderOutput();

  expect(output).toMatchSnapshot();
});
