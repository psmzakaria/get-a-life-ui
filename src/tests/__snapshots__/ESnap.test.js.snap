// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the create component of the app  1`] = `
<div
  className="esnap-container-size"
>
  <Modal
    centered={true}
    closeIcon={true}
    closeOnDimmerClick={true}
    closeOnDocumentClick={false}
    dimmer={true}
    eventPool="Modal"
    onClose={undefined}
    open={undefined}
    trigger={
      <Card
        color="red"
      >
        <CardContent>
          <CardHeader>
            birthday
          </CardHeader>
          <CardMeta>
            <span>
              Host: 
              lala
            </span>
          </CardMeta>
          <CardDescription>
             
            <Icon
              as="i"
              name="list"
              size="small"
            />
             KIV - 
          </CardDescription>
        </CardContent>
        <CardContent
          extra={true}
        />
      </Card>
    }
  >
    <ModalHeader>
      birthday
    </ModalHeader>
    <ModalContent>
      <ModalDescription>
        lala
        <br />
        <strong>
           Description:
        </strong>
         
        <br />
        <br />
        <Button
          as="button"
          floated="right"
          onClick={[Function]}
          role="button"
          type="submit"
        >
          Accept
        </Button>
        <Button
          as="button"
          floated="right"
          role="button"
          type="submit"
        >
          Reject
        </Button>
      </ModalDescription>
    </ModalContent>
  </Modal>
</div>
`;
